src/microbit/pxt_flasketuten_peker_paa/lesson.yml

```diff
@@ -0,0 +1,5 @@
+level: 2
+tags:
+  topic: [block_based]
```

Comment:
`electronics`, `animation` og `game` passar her.

Reply:
OK. Eg er usikker på kva for tags som kan/bør blir brukt.

Reply:
Alle mulige tags ligger her: https://github.com/kodeklubben/oppgaver/blob/master/filtertags/keys.yml